### DISCUSSION TITLE
Translation consistency

### DISCUSSION THREAD

**Discussion by ssnor** [FreeTubeApp/FreeTube#2060] [General]
Hello FreeTube devs and users!

I'm doing some translation for the Swedish language, and I noticed some discrepancies between the style of the previously translated sections and the source text.
I can see that English FreeTube generally uses Camel Case in text, but the Swedish translation doesn't.
When I became unsure I followed the precedent in Swedish when I translated.
Are there any "design document" or some statement on consistency between translations?
I admit that Camel Case is very uncommon to see in Swedish, so this might be a case of cultural adaptation on the previous translator.

I would also like to know how literal the project wishes for the translations to be?
For example, when I tried to translate "Hide Side Bar Labels" it felt more natural to translate something like "Hide Side Bar Text When Contracted", but it felt like altering the original a bit too much.
How much alteration is acceptable? You have to do some modification to the text to be properly translated without sounding like  a machine translation.

I hope this is the proper place to open this discussion, I tried to search if this has been discussed before but found nothing on Weblate, Issues or Discussions.

Regards
ssnor

**Comment by PrestonN** [comment]
Hey there!

Translations is definitely an odd one for me that's for sure. While I would love to have specific design standards for translations within FreeTube, there really isn't a good way for myself or others on the team to enforce them. I don't speak many of the languages that we offer so I have to trust that the translators are giving their best judgements when it comes to how certain text gets translated. While I'm available for questions if there are any, I'll have to trust that what makes more sense in your language is the better approach.

This is a fine spot for asking further questions if you have any. If translators feel like they'd work better having a Matrix chat room dedicated to translations, then I can open that up as well, though I haven't heard too much about wanting something like that.
